project/cmake/installdata/common/common.txt

```diff
@@ -3,11 +3,14 @@ sounds/*
 system/keymaps/*
 system/library/*
 system/players/VideoPlayer/etc/*
+system/python/*
 system/shaders/*
 system/settings/*
 userdata/*
 system/addon-manifest.xml
 system/colors.xml
+system/IRSSmap.xml
 system/peripherals.xml
 system/playercorefactory.xml
+system/X10-Lola-IRSSmap.xml
```

Comment:
I don't know if they are used at all under linux. I just want that <code>diff -qr autotools/ cmake/ | sort</code> returns a nice *nada*. :wink: